test/Reflection/FileReflectionTest.php

```diff
@@ -18,14 +18,6 @@
  */
 class FileReflectionTest extends \PHPUnit_Framework_TestCase
 {
-    public function testFileConstructor()
-    {
-        $filePath = __DIR__ . '/../../vendor/zendframework/zend-version/src/Version.php';
-        require_once $filePath;
-        $reflectionFile = new FileReflection($filePath);
-        $this->assertEquals(get_class($reflectionFile), 'Zend\Code\Reflection\FileReflection');
-    }
```

Comment:
I've found many zend-code tests do this kind of assertions about if the constructor don't throw any exception.

However rests of tests do the same but specific to the intended behavior. So I did remove this because its covered by the rest of tests.